tests/functional/workloads/cnv/test_vm_snapshot_clone.py

```diff
@@ -0,0 +1,82 @@
+import logging
+from ocs_ci.framework.pytest_customization.marks import workloads, magenta_squad
+from ocs_ci.framework.testlib import E2ETest
+from ocs_ci.helpers.cnv_helpers import cal_md5sum_vm, run_dd_io
+from ocs_ci.ocs import constants
+
+log = logging.getLogger(__name__)
+
+
```

Comment:
please add polarion id

Reply:
@avd-sagare please create the test in polarion